xml/Microsoft.Extensions.Logging.Abstractions.Internal/NullScope.xml

```diff
@@ -48,7 +48,7 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Performs application-defined tasks associated with freeing, releasing, or resetting unmanaged resources.</summary>
         <remarks>To be added.</remarks>
         <inheritdoc />
```

Comment:
@dotnet/docs this is the first time I see `<inheritdoc />` in dotnet-api-docs. What is it for?

Reply:
This enables a class to inherit the XML comments.  See here: https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/inheritdoc

Reply:
I should've clarified my question: does it actually work in dotnet-api-docs? I thought the purpose of this was for triple slash comments to be automatically inherited when building, but I didn't know they could also have a job in the xmls of this repo.